tutorials/plot_point_spread.py

```diff
@@ -0,0 +1,147 @@
+
+fname_evoked = osp.join(data_path, 'MEG', 'sample',
+                        'sample_audvis-ave.fif')
+
+subject_dir = osp.join(data_path, 'subjects/')
+
+###############################################################################
+# Load and process MEG data
+fwd = mne.read_forward_solution(fname_fwd, force_fixed=True,
+                                surf_ori=True)
+inv_op = read_inverse_operator(fname_inv)
+
+# We need the epochs data to compute the covariance, so we manually compute the
+# evoked data by averaging the epochs.
+evoked_ff = mne.read_evokeds(fname_evoked, condition=0, baseline=(None, 0))
+
+raw = mne.io.RawFIF(osp.join(data_path, 'MEG', 'sample',
+                             'sample_audvis_raw.fif'))
+events = mne.find_events(raw)
+event_id = {'Auditory/Left': 1, 'Auditory/Right': 2}
+epochs = mne.Epochs(raw, events, event_id, tmin=evoked_ff.times[0],
+                    tmax=evoked_ff.times[-1], baseline=(None, 0),
+                    preload=True)
+evoked = epochs.average()
+
+labels = mne.read_labels_from_annot('sample', subjects_dir=subject_dir)
+label_names = [l.name for l in labels]
+nlabs = len(labels)
+
+###############################################################################
+# Estimate noise covariance from baseline
+cov = mne.compute_covariance(epochs, tmin=None, tmax=0.)
+
+###############################################################################
+# Simulate known signals at two regions
+
+# Generate a known signal to corrupt with point spread
+idx = label_names.index('inferiorparietal-lh')
+signal = np.zeros((nlabs, T))
+signal[idx, :] = 1e-7 * np.sin(5 * 2 * np.pi * times)
+idx = label_names.index('rostralmiddlefrontal-lh')
+signal[idx, :] = 1e-7 * np.sin(7 * 2 * np.pi * times)
+
+###############################################################################
+# Create mask for each label so that the signal is only set at the center
+# vertex.
+hemi_to_ind = {'lh': 0, 'rh': 1}
+for i, label in enumerate(labels):
+    # Labels need values to use center_of_mass function
+    labels[i].values.fill(1.)
+    surf_vertices = fwd['src'][hemi_to_ind[label.hemi]]['vertno']
+    restrict_verts = np.intersect1d(surf_vertices, label.vertices)
+    com = labels[i].center_of_mass(subject='sample',
+                                   subjects_dir=subject_dir,
+                                   restrict_vertices=restrict_verts,
+                                   surf='white')
+    # Get center of vertex index in Label's vertix list
+    cent_idx = np.where(label.vertices == com)[0][0]
+
+    # Mask out all vertices except the center.
+    labels[i].values.fill(0.)
+    labels[i].values[cent_idx] = 1.
+
+###############################################################################
+# Create SourceEstimate object with known signal at center vertices of two
+# labels.
+stc_gen = simulate_stc(fwd['src'], labels, signal, times[0],
+                       times[1] - times[0], value_fun=lambda x: x)
```

Comment:
The actual times don't matter since it's fictional data. I personally like `linespace` better than `arange`, but happy to change if others feel differently. I forgot that I had made the `dt` variable in line 112, so I've changed `times[1]-times[0]` to `dt` there.